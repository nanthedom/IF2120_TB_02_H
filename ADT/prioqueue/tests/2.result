ADT/prioqueue/tests/2.in ADT/prioqueue/tests/2.out: TRUE
