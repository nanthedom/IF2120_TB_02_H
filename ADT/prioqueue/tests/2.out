4 200
3 100
2 210
#
